ConfigurationSystem/Client/Utilities.py

```diff
@@ -274,8 +274,7 @@ def addToChangeSet(entry, changeSet):
         if submissionMode == "Unknown" and newSubmissionMode:
           addToChangeSet((ceSection, 'SubmissionMode', submissionMode, newSubmissionMode), changeSet)
 
-        queues = ceInfo['Queues'].keys()
-        for queue in queues:
+        for queue in ceInfo['Queues'].keys():
```

Comment:
```suggestion
        for queue in ceInfo['Queues']:
```
actually 
```
    for queue, queueInfo in ceInfo['Queues'].items():
```
and dropping the next line, right?

Reply:
Yeah that can be done. In general I've avoided stylistic improvements as this PR is already larger than is reasonable.